docs/Example Job Specs/large-response-job.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: nodes.liquid
+date: Last Modified
+title: 'MultiWord Example Job Spec'
+permalink: "docs/example-job-spec-large/"
+---
+
+This is an example v1 job spec for returning multiple responses in 1 Chainlink API Call. 
+
+```JSON
+{
```

Comment:
Wait, don't we need to use the TOML version of the specs from now on?

Reply:
No we do, the TOML version of this job spec isn't ready. We will have to convert it to TOML laster